tests/conftest.py

```diff
@@ -189,7 +237,7 @@ def my_check_output(cmd):
         yield check_output
 
 
-@pytest.fixture(scope="session", autouse=True)
```

Comment:
Why this change? Was this fixture being applied too broadly?

Reply:
Any code that used `requests` had some bits several layers down the stack patched with a mock, because `requests` uses netrc. It was even worse than that sounds, because the resulting error was swallowed so I had no idea why my request was failing for quite a while. :woozy_face:

Reply:
ugh